doc/release-notes.rst

```diff
@@ -209,6 +209,7 @@ Notable Changes
 * msgr: fix RESETSESSION bug (#10080 Greg Farnum)
 * objectstore: deprecate collection attrs (Sage Weil)
 * osd, mon: add checksums to all OSDMaps (Sage Weil)
+* osd, mon: honor new 'norebalance' osdmap flag
```

Comment:
drop this hunk.  the release-notes are written after the next release is frozen; this is the last release. we don't know when this will appear yet..